src/Pages/Dashboard/historyChart.jsx

```diff
@@ -39,7 +39,7 @@ class Chart extends PureComponent {
                     >
```

Comment:
This file should just be deleted. Your work supersedes anything in this.

Reply:
Wow I didn't know the file was not used at all.